checkstyle.xml

```diff
@@ -44,19 +44,6 @@
         </module>
         <module name="NeedBraces"/>
         <module name="LeftCurly"/>
-        <module name="RightCurly">

```

Comment:
Why did you remove this?

Reply:
I removed it - <module name="RightCurly"> because when I tried to build the project (mvn clean package) I got an error -  '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks : if/else-if/else, to/in
hile or try/catch/finally). [RightCurlySame] it shows in Mapper, i tried to fix it, but i failed so i deleted it

Reply:
I see that your checkstyle plugin validates the compilation output as well. 
You can limit the scan only to source directories by modifying plugin's configuration in `pom.xml`:
```xml
<configuration>
	<sourceDirectories>
		<sourceDirectory>${project.build.sourceDirectory}</sourceDirectory>
		<sourceDirectory>${project.build.testSourceDirectory}</sourceDirectory>
	</sourceDirectories>
	<configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
	<consoleOutput>true</consoleOutput>
	<failsOnError>true</failsOnError>
	<linkXRef>false</linkXRef>
</configuration>
```

Reply:
Please add changes to `pom.xml`